app/javascript/packs/application.js.erb

```diff
@@ -0,0 +1,31 @@
+import "core-js/stable"
+import "regenerator-runtime/runtime"
+import RailsUJS from "@rails/ujs";
+import * as ActiveStorage from "@rails/activestorage";
+
+import "jquery-ui";
+import Listeners from "../listeners";
+import AjaxMixpanelEvents from "../lib/ajax_mixpanel_events";
+
+// Setting globally isn't very webpack-y,
+// but we have things (views, test runner) in the app that require jquery on the window object.
+window.jQuery = $;
+window.$ = $;
+
+RailsUJS.start();
```

Comment:
In https://github.com/rails/rails/tree/main/actionview/app/assets/javascripts they seem to recommend

```
import Rails from "@rails/ujs"
Rails.start()
```

Any reason to do Rails vs. RailsUJS?